src/unifiedConfig.json

```diff
@@ -0,0 +1,1306 @@
+[
+	{
+		"key": "prettier.general.section",
+		"title": "General",
+		"type": "section",
+		"children": [
+			{
+				"key": "prettier.module.path",
+				"title": "Prettier Module",
+				"type": "path",
+				"allowFiles": false,
+				"allowFolders": true,
+				"default": true,
+				"config": {
+					"description": "Set a folder for the Prettier module. If not set, it defaults to the build-in Prettier or the module in the project.",
+					"placeholder": "e.g. /opt/homebrew/lib/node_modules/prettier"
+				},
+				"configWorkspace": {
+					"description": "Set a folder for the Prettier module. If not set, it defaults to the global settings or the module in the project.",
+					"placeholder": "Global Setting"
+				}
+			},
+			{
+				"key": "prettier.debug.logging",
+				"title": "Log Debug Information",
+				"description": "Send debug information to the Extension Console.",
+				"config": {
+					"type": "boolean",
+					"default": false
+				},
+				"configWorkspace": {
+					"type": "enum",
+					"radio": false,
+					"values": ["Global Setting", "Enabled", "Disabled"],
+					"default": "Global Setting"
+				}
+			}
+		]
+	},
+	{
+		"key": "prettier.default-config.section",
+		"title": "Prettier Configuration",
+		"description": "Applied to remote files or when no Prettier configuration file exists.",
+		"type": "section",
+		"children": [
+			{
+				"key": "prettier.default-config.printWidth",
+				"title": "Print Width",
+				"description": "Specify the line length that the printer will wrap on.",
+				"link": "https://prettier.io/docs/en/options.html#print-width",
+				"type": "number",
+				"min": 0,
+				"config": {
+					"placeholder": "80",
+					"default": 80,
+					"required": true
+				},
+				"configWorkspace": {
+					"placeholder": "Global Setting",
+					"default": null
+				}
+			},
```

Comment:
This part here for example, replaces these two parts in the extension.json:

_config:_

https://github.com/alexanderweiss/nova-prettier/blob/deb51976ae4d0d48766c69dc8757135b0b3fb138/prettier.novaextension/extension.json#L73-L86

_configWorkspace:_
https://github.com/alexanderweiss/nova-prettier/blob/deb51976ae4d0d48766c69dc8757135b0b3fb138/prettier.novaextension/extension.json#L364-L376

And this is analogue for all other keys.

Reply:
The syntax/logic is as follows:

```
{
  common keys: "shared between global and workspace config",
  
  config: {
      keys only used by the global config
  },
  configWorkspace: {
      keys only used by the workspace config
  }
}
```

PR #109 iterates through this JSON file and creates the `config.json` and `configWorkspace.json`. Both the contain all keys that are necessary for nova to create the settings page.